fay.cabal

```diff
@@ -1,5 +1,5 @@
 name:                fay
-version:             0.24
+version:             0.23.2.0
```

Comment:
This is sufficient since we didn't change fay's API, and then we don't need to update fay-base.